Dockerfile

```
@@ -1,6 +1,26 @@
+#
+# BUILDER
+#
+FROM docker.io/library/golang:1.21.1 as builder
```

```suggestion
FROM docker.io/library/golang:1.21.1 AS builder
```

No impact on the behavior, but IMHO looks better.